internal/templates/nginx/yamls/ingress.yaml

```diff
@@ -0,0 +1,81 @@
+{{- if .Values.app.isAccessible }}
+{{- if .Values.app.ingress.http }}
+{{- range $i, $deployment := .Values.app.deployments }}
+{{- if gt $deployment.routingSettings.weight 0.0}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ $.Values.app.name }}-{{ $i }}-http-ingress
+  annotations:
+    {{- if $.Values.ingressController.className }}
+    kubernetes.io/ingress.class: {{ $.Values.ingressController.className }}
```

Comment:
let's use `quote` function there
```
    kubernetes.io/ingress.class: {{ $.Values.ingressController.className | quote }}
```
we will fail to render a chart if className is  `true` or `1`. 
I'll be a valid k8s manifest but helm doesn't like such annotations/labels

Reply:
lets fix in another PR